lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -302,13 +297,7 @@ func (h *StorageHandlersImpl) VolumeStoresList(params storage.VolumeStoresListPa
 			})
 	}
 
-	resp := &models.VolumeStoresListResponse{
-		Stores: make(map[string]string),
-	}
-
-	for name, ds := range stores {
-		resp.Stores[name] = ds.String()
-	}
+	resp := &models.VolumeStoresListResponse{Stores: stores}
```

Comment:
Nevermind, I see you changed the cache to returning an array. Ignore this, commented before I made it all the way through the PR.